src/sonic-bgpcfgd/bgpcfgd/main.py

```diff
@@ -96,10 +96,14 @@ def do_work():
         managers.append(BfdMgr(common_objs, "STATE_DB", swsscommon.STATE_BFD_SOFTWARE_SESSION_TABLE_NAME))
 
     device_metadata = config_db.get_table("DEVICE_METADATA")
-    if "localhost" in device_metadata and "type" in device_metadata["localhost"] and device_metadata["localhost"]["type"] == "SpineRouter" and "subtype" in device_metadata["localhost"] and device_metadata["localhost"]["subtype"] == "UpstreamLC":
+    # Enable Prefix List Manager and AsPath Manager for UpperSpineRouter/UpstreamLC
+    if ("localhost" in device_metadata and "type" in device_metadata["localhost"] and
+        (device_metadata["localhost"]["type"] == "SpineRouter" and "subtype" in device_metadata["localhost"] and device_metadata["localhost"]["subtype"] == "UpstreamLC" or
+         device_metadata["localhost"]["type"] == "UpperSpineRouter")):
```

Comment:
could you please group spinerouter + upstreamLC checks with braces to improve readability on this?

Reply:
Updated